### ISSUE TITLE
Reworking alerts

### ISSUE THREAD

**Issue by vlab97** [xvitaly/ecasbot#199]
### Feature request

Предлагаю идею с переработкой функций "restalert", "delalert". Сейчас они работают в связке с командой /subscribe. В идеале их отделить на 2 отдельные команды. Чтоб каждый администратор мог настраивать лично под себя вид оповещений. 

Пример

/subscribe - подписка на жалобы и прочее (используется в групповом чате) (уже имеется)

/delalert - Подписка на оповещения про удаления сообщений (используется в групповом чате)

/restalert - Подписка на оповещения о наложении ограничений на новых пользователей (используется в групповом чате)

**Comment by xvitaly** [comment]
Да, это возможно хорошая функция для 2.0.0 ибо потребует серьезной переработки.

**Comment by vlab97** [comment]
> Да, это возможно хорошая функция для 2.0.0 ибо потребует серьезной переработки.

Может тогда весь функционал бота переработать? Чтоб каждый чат выбирал под себя параметры бота. Тот же самый выбор видов entries для блокировки, количество дней ограничения и прочее. Масштабно😁
